client/my-sites/checkout/checkout/index.jsx

```diff
@@ -468,14 +482,8 @@ export class Checkout extends React.Component {
 
 		const queryParam = displayModeParam ? `?${ displayModeParam }` : '';
 
-		if ( this.props.isEligibleForCheckoutToChecklist & ( ':receiptId' !== receiptId ) ) {
-			const destination = abtest( 'improvedOnboarding' ) === 'main' ? 'checklist' : 'view';
-
-			return `/${ destination }/${ selectedSiteSlug }${ queryParam }`;
-		}
-
-		if ( this.props.isJetpackNotAtomic ) {
-			return `/plans/my-plan/${ selectedSiteSlug }?thank-you&install=all`;
+		if ( this.props.isEligibleForSignupDestination & ( ':receiptId' !== receiptId ) ) {
```

Comment:
Just a tiny nit here, but does this one need to be a bitwise `&`, or can it be changed to a logical and `&&`?

Reply:
Thanks for the catch! It was a typo, fixed in [0a1153f](https://github.com/Automattic/wp-calypso/pull/34156/commits/0a1153f8189218e80b4af70c2218422eda38f6f9).